src/ngraph/frontend/onnx_import/op/round.cpp

```diff
@@ -0,0 +1,42 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "round.hpp"
+#include <cstddef>
+#include <cstdint>
+#include <memory>
+#include "default_opset.hpp"
```

Comment:
So you should include `ngraph/opsets/opset0.hpp` instead of `default_opset.hpp`.
And it could be better to move `round.hpp` header below libs headers but make sure if you really need more than `<memory>` here:)

Reply:
Done.